src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -188,9 +188,10 @@ CHIP_ERROR BindingManager::NotifyBoundClusterChanged(EndpointId endpoint, Cluste
         {
             if (iter->type == EMBER_UNICAST_BINDING)
             {
-                mPendingNotificationMap.AddPendingNotification(iter.GetIndex(), bindingContext);
+                error = mPendingNotificationMap.AddPendingNotification(iter.GetIndex(), bindingContext);
```

Comment:
Adding comment to prevent this PR from getting auto merged if someone else approves. There is something else that still needs to be fixed